cmd/clusterctl/pkg/client/config/providers_client.go

```diff
@@ -61,10 +61,10 @@ func newProvidersClient(reader Reader) *providersClient {
 
 func (p *providersClient) defaults() []Provider {
 
-	// clusterctl includes an hard-coded list of "default" providers in order to provide to the users the simplest
-	// out-of-box experience. Other providers can be added by using the clusterctl configuration file.
+	// includes a predefined list of Cluster API providers sponsored by SIG-cluster-lifecycle to provide users the simplest
```

Comment:
I think we lost clusterctl at the beginning of the line?

Reply:
😖